tests/integration/llm/prepare.py

```diff
@@ -509,6 +509,18 @@
     },
 }
 
+lmi_dist_aiccl_model_list = {
```

Comment:
we will need P4D to test this

Reply:
Yes. I added this as a todo in the PR description for tracking